app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    result = {}
+
+    def wrapper(*args: (int, tuple)) -> int:
+        if args in result:
+            print("Getting from cache")
+            return result[args]
```

Comment:
code duplicate move it after if

Reply:
live 10 and line 14 the same

Reply:
I don't really understand it, what is duplicated?

Reply:
https://github.com/mate-academy/py-cache-decorator/blob/master/checklist.md

Reply:
See duplicate and checklist
Not fixed